 framework: shorten wait time for shutdown time optimization

Default 500ms delay is waitting shutdown actions done in while cycle.
Shutdown actions include broadcast,nfc,radio,wifi and mountservice. The whole
actions are ordered serial operations.This delay is too long for broadcast/nfc
/radio/wifi/mountservice shutdown. Decrease the delay will add mActionDoneSync
lock time and while cycle times, so this delay also can't be too short.
Set half will have 0.5~0.7s benifit for shutdown time, total time from 2.4s to
 1.7~1.9s, which is comparable with ref device.

diff --git a/services/core/java/com/android/server/power/ShutdownThread.java b/services/core/java/com/android/server/power/ShutdownThread.java
index 5f01e30..4795e97 100644
--- a/services/core/java/com/android/server/power/ShutdownThread.java
+++ b/services/core/java/com/android/server/power/ShutdownThread.java
@@ -72,7 +72,7 @@
 public final class ShutdownThread extends Thread {
     // constants
     private static final String TAG = "ShutdownThread";
-    private static final int PHONE_STATE_POLL_SLEEP_MSEC = 500;
+    private static final int PHONE_STATE_POLL_SLEEP_MSEC = 250;
     // maximum time we wait for the shutdown broadcast before going on.
     private static final int MAX_BROADCAST_TIME = 10*1000;
     private static final int MAX_SHUTDOWN_WAIT_TIME = 20*1000;
